packages/vue/src/components/organisms/SfTable/_internal/SfTableRow.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  name: "SfTableRow",
+  mounted() {
+    this.$parent.updateColumnsCount(
```

Comment:
sometimes people might want to wrap columns into links etc, better do a check if there is parent and catch error OR use provide/inject

Reply:
wrap td / th into links it's a semantic way?

Reply:
but sure :) provide/inject is ok 👍

Reply:
@przspa expect unexpected, here people might not be aware that its semantically correct table ;)